src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -30,6 +34,61 @@ interface HookValidatedTextInputPropTypes extends TextInputProps {
   warning?: string;
 }
 
+export const HookPasswordValidatedInput = ({
+  ariaLabel,
+  placeholder,
+  dataTestId,
+  value,
+  ouiaId,
+  stepValidation,
+  fieldName,
+  onChange,
+  warning = undefined,
+}: HookValidatedTextInputPropTypes) => {
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const togglePasswordVisibility = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+
+  return (
+    <>
+      <InputGroup>
+        <InputGroupItem isFill>
+          <HookValidatedInput
+            isRequired
+            type={isPasswordVisible ? 'text' : 'password'}
+            ouiaId={ouiaId || ''}
+            data-testid={dataTestId}
+            aria-label={ariaLabel || ''}
```

Comment:
There are duplicate aria labels in the props - `aria-label` and `ariaLabel`. `isDisabled` prop is missing. `warning` is missing also.

Reply:
I will add them as part of rest., what do you think?

Reply:
I will pass some of the field as rest...